docs/recipes/lighthouse-plugin-example/readme.md

```diff
@@ -0,0 +1,17 @@
+# Lighthouse plugin recipe
+
+## Contents
+- `package.json` - declares the plugin's entry point (`plugin.js`)
+- `plugin.js` - instructs Lighthouse to run the plugin's own `preload-as.js` audit; describes the new category and its details for the report
+- `audits/preload-as.js` - the new audit to run in addition to Lighthouse's default audits
+ 
+## To run
+
+- Install somehow.
```

Comment:
we could nuke private and publish this. sets the convention of folks publishing with the `lighthouse-plugin-` prefix.

or w/e. i'm chill.